src/components/ImagesList.tsx

```diff
@@ -0,0 +1,14 @@
+import { ImagesInfo } from './ImagesInfo';
+
+export const ImagesList: React.FC <{
+  images: string[];
+  pass: { itemWidth: number };
+}> = ({ images, pass }) => (
+  <>
+    {
+      images.map((image) => (
+        <ImagesInfo image={image} pass={pass} />
```

Comment:
don't forget to add `key` when you're rendering the list of elements
```suggestion
        <ImagesInfo key={image} image={image} pass={pass} />
```